CMakeLists.txt

```
@@ -156,7 +156,15 @@ include_directories(
 option(BUILD_PROOF_OF_SPACE_STATICALLY "Build ProofOfSpace target statically" OFF)
 IF (BUILD_PROOF_OF_SPACE_STATICALLY)
   message("Statically build ProofOfSpace")
-  target_link_libraries(ProofOfSpace -static -Wl,--whole-archive -lrt -lpthread -Wl,--no-whole-archive)
+  target_link_libraries(ProofOfSpace PUBLIC -static -Wl,--whole-archive -lrt -lpthread -Wl,--no-whole-archive)
+ENDIF()
+
+option(BUILD_STATIC_CHIAPOS_LIBRARY "Build chiapos library statically" OFF)
```

```suggestion
option(BUILD_STATIC_CHIAPOS_LIBRARY "Build chiapos static library (verify-only)" OFF)
```

Nit pick, a library is "static" and you can link against it "statically".